### ISSUE TITLE
L Nav Accessibility

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#8145]
**Design development for the left Nav upgrade was completed in [7212]**

## User Story or Problem Statement
As a veteran, I need the left menu elements assist me in navigating the website so I can find what I am looking for more easily. 

## Goal
Ensure Left Nav is 508 compliant

## Objectives or Key Results this is meant to further
OBJECTIVE 3: Deliver a usable and accessible experience

---
## Tasks
- [ ] Use aria label (`aria-label="Sentence case words`) to ensure section labels can be read as words  
- [ ] Ensure the side navigational system is keyboard accessible 
- [ ] Ensure content structure may be navigated by tab order
- [ ] Include markup for properly naming this <nav> element with a descriptive name (example = "Services and locations") which can be read aloud for screen readers.
- [ ]  Use aria labels or titles to make landmarks unique. See [#4215](https://github.com/department-of-veterans-affairs/va.gov-team/issues/4215) for recommendations. 
- [ ] Make sure the divider lines are either hidden programmatically or are borders created with CSS.
- [ ] Each section should have a properly nested heading with relationships to the previous parent using ARIA
- [ ] Incorporate proper semantic markup. See [#3441] for recommendations.
- [ ] Convey the menu structure with a list to allow assistive technologies to announce the number of items in the menu and provide corresponding navigation functionality. 
- [ ] Identify the menu (using <nav> element) to allow users access to the menu directly. 
- [ ] Use a heading (aria-label, or aria-labelledby) to label the menu to make it easier to find understand and  distinguish between multiple menus on a web page. 
- [ ] Use markup to indicate the current item of a menu, such as the current page on a website, to improve orientation in the menu.
- [ ] The menu size adapts to varying text sizes, to accommodate languages with longer words and people who need larger text.

### Active/On State
- [ ] Announce "on state" (`aria-current="page" attribute on the current <a> tag)

### Focus state
- [ ] Focus must go to the top of the page for each page

### Open state
- [ ] Include an indicator of current page for screen readers (aria-current="page" on the current page)
- [ ] Turn off pointer-events and default the cursor on the aria-current="page" breadcrumb

 
### Mobile
- [ ] Should use good accordion ARIA markup for the mobile "On this section" button
> This includes aria-expanded="true | false" and ideally an aria-controls="UNIQUE_ID" that points to the navigation container. aria-controls only works for JAWS, but it's worth adding as a progressive enhancement.

## Acceptance Criteria

**Given https://staging.va.gov/pittsburgh-health-care/ is being accessed by a user with assistive technology**
- [ ] When a screenreader user clicks an items from the left nav, there is an indicator of the current page. 
- [ ] Users should be able to tab through each link.
- [ ] A yellow halo focus state is present in both hover and active states. 
- [ ] "on state" is announced to screen readers
- [ ] Use aria label (`aria-label="Sentence case words`) to ensure section labels can be read as words  
- [ ] Elements have a descriptive name which can be read aloud
- [ ] Screen readers read the words of sections (rather than spelling out letters.)
- [ ] Sections have properly nested headings
- [ ] User can select menu item using keyboard space bar or enter key. 
- [ ] Screen reader users can hear the content structure with headings to navigate the page content 
- [ ] Web page has an acceptable color contrast ratio. 
- [ ] No information provided in the page is only conveyed through color.
- [ ] Actionable links have visual cues
- [ ] Focus is always visible when moving through the page with the keyboard. 
- [ ] Site is free of keyboard traps
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)## User Story or
